crates/holochain-conductor-runtime-ffi/scripts/build-single-target.sh

```diff
@@ -1,11 +1,13 @@
 #/usr/bin/env bash
```

Comment:
Build rust library and bindings for a specified target arch. This was actually what I wanted out of the build-debug.sh script.